src/web/demo.rkt

```diff
@@ -518,7 +519,8 @@
       (define formula-str (hash-ref post-data 'formula))
       (define formula (read-syntax 'web (open-input-string formula-str)))
       (define seed (hash-ref post-data 'seed))
-      (define hash (sha1 (open-input-string formula-str)))
+      (define hash (sha1 (open-input-string 
```

Comment:
Thinking about this I wonder if I should hash the seed and when applicable the samples. 🤔. 
Hopefully hashing time doesn't blow up too much 😂.

Reply:
Hashing time will be super fast. I think instead of `string-append`, why not take the whole message you send to the worker thread, convert it to a string (with `~s`), hash that, and use that as the hash key?